src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -671,15 +685,46 @@ private static bool GetEnvironmentVariableAsBool(string name, bool defaultValue)
             return defaultValue;
         }
 
+        /// <summary>
+        /// Send module load telemetry as a metric.
+        /// For modules we send the module name (if allowed), and the version.
+        /// Some modules (CIM) will continue use the string alternative method.
+        /// </summary>
+        /// <param name="telemetryType">The type of telemetry that we'll be sending.</param>
+        /// <param name="moduleInfo">The module to report. If it is not allowed, then it is set to 'anonymous'.</param>
+        internal static void SendModuleTelemetryMetric(TelemetryType telemetryType, PSModuleInfo moduleInfo)
+        {
+            if (!CanSendTelemetry)
+            {
+                return;
+            }
+
+            // Package up the module name, version, and known tag as a metric.
+            try
+            {
+                string allowedModuleName = GetModuleName(moduleInfo.Name);
+                string allowedModuleVersion = allowedModuleName == Anonymous ? AnonymousVersion : moduleInfo.Version?.ToString();
+                string allowedModuleTag = moduleInfo.Tags.FirstOrDefault(t => s_knownModuleTags.Contains(t)) ?? NoTag;
```

Comment:
What happens in the future if we have more tags and a module has more than one?

Reply:
@SteveL-MSFT with this PR, we'll get the just one of the matches. I don't really have a good suggestion for that at present. I suppose we could add all the tags that match, and it shouldn't be adding that much to the payload.

Reply:
i was able to collect all the tags which we're looking for and report on them